dandi/cli/command.py

```diff
@@ -186,6 +186,11 @@ def main(log_level, pdb=False):
 
         etelemetry.check_available_version("dandi/dandi-cli", __version__, lgr=lgr)
     except Exception as exc:
+        # This magical undocumented env var would allow to overcome some internal
+        # or external issues by merely disclosing its availability to effected users,
+        # while mandating users to update to the most recent version of the client
+        if not bool(os.environ.get("DANDI_ALLOW_OUTDATED", None)):
+            raise
         lgr.warning(
             "Failed to check for a more recent version available with etelemetry: %s",
```

Comment:
this message seems more generic than perhaps we should use? you can use the flag `raise_exception=True` in the call to `check_available_version` and will return a `RuntimeError` if a bad version is running.

Reply:
`RuntimeError`?  submitted https://github.com/sensein/etelemetry-client/pull/23 

as for `raise_exception=True` - I should add it indeed but then, depending on etelemetry version I would not be reliably tell apart either it is outdated or some other exception has happened (there is sufficient amount of code outside of try/except in that function).  I don't think we would like to stop on any exception etelemetry is throwing, but on that `BadVersionException` - so I will wait on destiny of that PR